app/admin_resources.py

```diff
@@ -20,7 +20,7 @@
 import mimetypes
 
 import config
-import const
+from utils import const
```

Comment:
Did you intend to include all these changes related to const.py in this PR (the title says it's just for moving the data files)?

Reply:
@nworden Initially in PR #450 ( closed ) I moved files related to handlers and utils into respective directories. This PR actually reverts them to app/ and copies only contents related to app/data/

Reply:
> Did you intend to include all these changes related to const.py in this PR (the title says it's just for moving the data files)?

@nworden It was actually the part of handlers which were supposed to be moved from app directory. Now since handlers are kept under app/ we can keep const.py here itself.

Reply:
> The [diffs I'm looking at](https://github.com/google/personfinder/pull/578/files) are your current PR compared to our master branch, and there are a lot of changes unrelated to the data files. In particular, in many places it changes imports of `const` from `import const` to `from utils import const` (along with changes to sys.path and the django.po files).

@nworden I have reverted that change to import const